### ISSUE TITLE
await-async-query not waiting when method is called from wrapper

### ISSUE THREAD

**Issue by victorandcode** [testing-library/eslint-plugin-testing-library#113]
Hello,

I've been trying to integrate the rule `await-async-query` with a React project but it doesn't seem to be working as expected. Here's how to reproduce the problem:

```javascript
import React from "react";
import { render, screen } from "@testing-library/react";

const Foo = () => <div>Hello</div>;

it("should fail rule", () => {
  render(<Foo />);

  screen.findByText('Hello'); // DOES NOT FAIL await-async-query even though it should
  
  expect(1).toBe(1)
});
```

.eslintrc
```
{
    "env": {
        "browser": true,
        "commonjs": true,
        "es6": true,
        "jest": true
    },
    "parserOptions": {
        "ecmaVersion": 2018,
        "sourceType": "module",
        "ecmaFeatures": {
            "jsx": true
        }
    },
    "plugins": ["testing-library"],
    "rules": {
        "testing-library/await-async-query": "error",
        "testing-library/await-async-utils": "error",
        "testing-library/no-await-sync-query": "error",
        "testing-library/prefer-explicit-assert": "error",
        "testing-library/prefer-presence-queries": "error",
        "testing-library/prefer-wait-for": "error"
    }
}
```

This is strange for me because the rule is recommended to work with the react version of testing-library which uses either screen or wrapper instead of the direct export of find* methods.

I also noticed that the rule's selector used to work with this example previously but was changed in https://github.com/testing-library/eslint-plugin-testing-library/commit/60862ad6e2cee2106ba6ffb95d7707048ad21d22#diff-b939ebc192e4cb00915398cde14390f7R29

Please let me know if this rule is not intended to be used this way.

Thanks for the help :)

**Comment by Belco90** [comment]
Hello Victor, thanks for opening an issue for this! You are right: this rule should take care of queries from `screen`, but we didn't update that when it was released. It should be easy to fix.

**Comment by victor-cordova** [comment]
No problem. I'd like to take a look at it then :) I'll create a PR when ready

**Comment by Belco90** [comment]
That would be amazing, thanks!

**Comment by victorandcode** [comment]
@Belco90 PR ready for review :smile:
